core/server/common/src/main/java/alluxio/master/journal/raft/RaftJournalConfiguration.java

```diff
@@ -53,6 +58,28 @@ public static RaftJournalConfiguration defaults(ServiceType serviceType) {
         .setPath(new File(JournalUtils.getJournalLocation().getPath()));
   }
 
+  /**
+   * @param clusterAddresses addresses of all nodes in the Raft cluster
+   * @return true if the cluster addresses contain the local IP, false otherwise
+   */
+  private boolean checkContainLocalIp(List<InetSocketAddress> clusterAddresses) {
+    String clusterNodeIp;
```

Comment:
nit: can we define this variable inside the for-loop?
we don't need it outside the for-loop, right?